modules/schematics-policy/README.md

```diff
@@ -0,0 +1,3 @@
+# Schematics Agent Policy
+
+This submodule creates and configures the Schematics Agent Policy. More info on working with the agent policies can be found [here](https://cloud.ibm.com/docs/schematics?topic=schematics-policy-manage&interface=ui).
```

Comment:
This readme is for a module and therefore should have all the associated info on module (usage, required permissions, the terradocs info etc)